weblate/static/style-bootstrap.css

```diff
@@ -1617,10 +1617,18 @@ input.color_edit:checked + label {
   color: #2a3744;
 }
 
+table:has(#machinery-translations) {
+  table-layout: fixed;
+}
+
 .machinery-text {
   word-wrap: anywhere;
 }
 
+.machinery-diff {
```

Comment:
it might even be enough to just add this CSS rule...

Reply:
actually, the CSS rule that would work alone would be:
`word-break: break-word`